### PR TITLE
Add Coala for PEP8

### PR DESCRIPTION
**Pull request by ghoshbishakh**

### CODE REVIEW

**Comment by ghoshbishakh** [comment]
@nkprince007 take a look if all configs are OK

File: .coafile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[Default]
+bears = PEP8Bear
+files = festflow/admin.py,festflow/apps.py,festflow/forms.py,festflow/__init__.py,festflow/models.py,festflow/pipeline.py,festflow/tests.py,festflow/urls.py,festflow/utils.py,festflow/views.py,festcms/__init__.py,festcms/urls.py,festcms/wsgi.py
```
Comment (nkprince007) [inline .coafile]: Why not just `*.py`?

Comment (nkprince007) [inline .coafile]: And `ignore=blob_pattern` for rejecting files, if any?

File: requirements.txt [hunk 2]
```diff
@@ -1,12 +1,87 @@
...
+requests==2.13.0
 requests-oauthlib==0.6.2
+restructuredtext-lint==0.17.2
+rstcheck==2.2
+safety==0.5.1
+scspell3k==2.1
 six==1.10.0
-django-ckeditor==5.1.1
-django-forms-builder==0.13.0
+smmap2==2.0.1
```
Comment (nkprince007) [inline requirements.txt]: You don't need all these requirements, you could just use the `cib` and `coala` package
and run the script `cib install -c .coafile` @ghoshbishakh

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "3.5"
+# command to install dependencies
+install: "pip install -r requirements.txt"
+# command to run tests
+script: coala --ci
```
Comment (nkprince007) [inline .travis.yml]: new line please

**Comment by nkprince007** [comment]
And please try to add `appveyor` too, we might need to test on windows too, although this is only server stuff and is mostly run on linux only.

Mind the mac specific build for travis too.

**Comment by nkprince007** [comment]
If you want me to work on this, I'll could fix this up for you. 😄

**Comment by ghoshbishakh** [comment]
@nkprince007 no I want you to review my work on this one 😉

**Comment by nkprince007** [comment]
BTW please unhook the travis build until this work is complete, so that other PR's could remain unaffected.

**Comment by ghoshbishakh** [comment]
@nkprince007 let it be. We will just ignore travis.

**Comment by nkprince007** [comment]
But why?

**Comment by ghoshbishakh** [comment]
@nkprince007 what why?

**Comment by ghoshbishakh** [comment]
@nkprince007 were you working on it?

**Comment by nkprince007** [comment]
Closed 😝

**Comment by nkprince007** [comment]
Nope.

**Comment by ghoshbishakh** [comment]
@nkprince007 nah just resetted it. Will repoen in a bit

**Comment by nkprince007** [comment]
Ok.

**Comment by ghoshbishakh** [comment]
@nkprince007 review plase 😄

**Comment by ghoshbishakh** [comment]
@nkprince007 can we merge this fast so that we can check code standards for @akshatnitd ?

File: requirements.txt [hunk 4]
```diff
@@ -1,12 +1,37 @@
+appdirs==1.4.0
+autoflake==0.6
+autopep8==1.2
```
Comment (nkprince007) [inline requirements.txt]: why all these stuff? Just add `coala` and `coala-bears` as requirements.

Comment (ghoshbishakh) [inline requirements.txt]: Using cib

Comment (ghoshbishakh) [inline requirements.txt]: @nkprince007 just using coala-bears creates even more dependencies than these.

**Comment by nkprince007** [comment]
I'll modify this PR and merge it once the tests go green, hope you don't mind.

**Comment by ghoshbishakh** [comment]
@nkprince007 how can tests go green?

File: requirements.txt (hunk 4 above)
Comment (nkprince007) [inline requirements.txt]: Yeah, we don't need these requirements, although, we could just add `cib` and add a script line `cib install -c .coafile`

**Comment by nkprince007** [comment]
By adding another commit that fixes the errors shown. 😉

File: requirements.txt (hunk 4 above)
Comment (ghoshbishakh) [inline requirements.txt]: @nkprince007 okay let us make a requirements-dev.txt

**Comment by ghoshbishakh** [comment]
@nkprince007 we will need to fix all pep8 and all stuff to make it green. That will be an ongoing process

**Pull request merged**